covid19-dashboard/main.py

```diff
@@ -11,181 +11,110 @@
-    }
-
-    print("Getting perMonth Data")
-    monthly = {
-        'cases': covid.get_cases_for_given_range_alone(region=region, most_recent_date=date, time_delta=30).to_dict(),
-        'deaths': covid.get_deaths_for_given_range_alone(region=region, most_recent_date=date, time_delta=30).to_dict()
-    }
-    print("Getting monthlyPerCapita Data")
-    monthlyPerCapita = {
-        'cases': (covid.get_cases_difference_per_capita(region=region, most_recent_date=date, time_delta=30)).to_dict(),
-        'deaths': (
-            covid.get_deaths_difference_per_capita(region=region, most_recent_date=date, time_delta=30)).to_dict()
-    }
-
-    print("Getting weekIncrease Data")
-    monthlyIncrease = {
-        'cases': (covid.get_cases_increase_pct_for_given_dates(region=region, most_recent_date=date,
-                                                               time_delta=30)).to_dict(),
-        'deaths': (covid.get_deaths_increase_pct_for_given_dates(region=region, most_recent_date=date,
-                                                                 time_delta=30)).to_dict()
-    }
-
-    print("Getting absoluteCumulative Data")
-    absoluteCumulative = {
-        'cases': covid.get_cumulative_cases_for_given_date(region=region, date=date).to_dict(),
-        'deaths': covid.get_cumulative_deaths_for_given_date(region=region, date=date).to_dict()
-    }
-
-    print("Getting cumulativePerCapita Data")
-    cumulativePerCapita = {
-        'cases': (covid.get_cumulative_cases_per_capita_for_given_date(region=region, date=date)).to_dict(),
-        'deaths': (covid.get_cumulative_deaths_per_capita_for_given_date(region=region, date=date)).to_dict()
-    }
-
-    return {
-        "cumulativeTimeSeries": cumulativeTimeSeries,
-        "daily": daily,
-        "dailyPerCapita": dailyPerCapita,
-        "dailyIncrease": dailyIncrease,
-        "weekly": weekly,
-        "weeklyPerCapita": weeklyPerCapita,
-        "weeklyIncrease": weeklyIncrease,
-        "monthly": monthly,
-        "monthlyPerCapita": monthlyPerCapita,
-        "monthlyIncrease": monthlyIncrease,
-        "absoluteCumulative": absoluteCumulative,
-        "cumulativePerCapita": cumulativePerCapita
-    }
+@cache.cached(timeout=50000)
+@app.route("/get-places")
+def get_places():
+    """Returns a dictionary of dcid->(name, type)
+    where name is the name of the region and type is the type of region."""
+    response: dict = send_request(req_url='https://api.datacommons.org/node/property-values',
+                                  req_json={"dcids": ['NYT_COVID19_County', 'NYT_COVID19_State'],
+                                            "property": "member"},
+                                  post=True,
+                                  compress=False,
+                                  api_key=API_KEY)
+
+    counties: dict = {county['dcid']: (county['name'], 'county') for county in response['NYT_COVID19_County']['out']}
```

Comment:
County. since this is the type of place, in schema it is "County", also for "State" below

Reply:
Fixed. Capitalized County and State.